Source/Fluxor/Fluxor.csproj

```diff
@@ -6,11 +6,6 @@
     <PackageTags>Redux Flux DotNet CSharp</PackageTags>
   </PropertyGroup>
 
-  <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netstandard2.1;netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```

Comment:
Why are netstandard 2.0 and 2.1 no longer supported?

Reply:
They are I just hoisted the dependencies up into your Directory.Build.props so you only have one spot to make changes going forward.

Reply:
Re: netstandard 2.0 and 2.1, do the Blazor libraries still compile with those there?

Reply:
Indeed.

![image](https://user-images.githubusercontent.com/1130210/161806069-ed9f3b72-dfad-4752-95cd-ac198c64308c.png)

![image](https://user-images.githubusercontent.com/1130210/161806939-a6644cae-f6bf-4c6b-9dc5-f427da415618.png)